src/Modules/Backend/Domain/Action/AbstractActionController.php

```diff
@@ -66,11 +77,63 @@ final public static function getActionSlug(): ActionSlug
 
     public function __invoke(Request $request): Response
     {
+        if ($this->shouldUserConfigure2FA($request)) {
+            $this->header->addFlashMessage(
+                new FlashMessage(
+                    'msg.2FAIsRequired',
+                    FlashMessageType::WARNING
+                )
+            );
+
+            return $this->redirectToUserPage($request);
+        }
+
         $this->execute($request);
 
         return $this->getResponse($request);
     }
 
+    private function redirectToUserPage(Request $request): Response
+    {
+        /** @var User $user */
+        $user = $this->tokenStorage->getToken()->getUser();
+
+        return new RedirectResponse(
+            UserEdit::getActionSlug()->generateRoute($this->router) . '/' . $user->getId()
```

Comment:
same comment with the slug, please use the proper way to generate urls

Reply:
fixed 0209df4dd4bbb314d59176aa104a996ba2275106